DigitalLearningSolutions.Web/Controllers/MyAccountController.cs

```diff
@@ -32,5 +31,33 @@ public IActionResult Index()
 
             return View(model);
         }
+
+        [HttpGet]
+        public IActionResult EditDetails()
+        {
+            if (!User.Identity.IsAuthenticated)

```

Comment:
Not sure how I missed it in the My account review, but we should be able to just use [Authorize] annotation instead of this. :)

Reply:
OK will update all the existing ones while I'm doing this ticket